src/core/main.cpp

```diff
@@ -453,7 +453,7 @@ void print_xlio_global_settings()
                       NEW_CONFIG_VAR_LOG_DETAILS);
     VLOG_PARAM_STRING("Log Colors", safe_mce_sys().log_colors, MCE_DEFAULT_LOG_COLORS,
                       NEW_CONFIG_VAR_LOG_COLORS,
-                      safe_mce_sys().log_colors ? "Enabled " : "Disabled");
+                      safe_mce_sys().log_colors ? "true" : "false");
```

Comment:
I'd suggest to add VLOG_PARAM_BOOL or VLOG_PARAM_BOOLEAN and hide the output presentation logic there. In this case it will be easier to change output, for example, make the first letter upper case.